src/core_plugins/input_control_vis/public/register_vis.js

```diff
@@ -48,6 +48,11 @@ function InputControlVisProvider(Private) {
     },
     requestHandler: 'none',
     responseHandler: 'none',
+    options: {
+      showIndexSelection: false,
```

Comment:
now that I see the `showTimePicker` option in markdown, I think this too should be used for input controls?

Reply:
input controls can be filtered by time